tests/ui/apps/Scroller/EditableItem/ScrollerEditableItem-View.js

```diff
@@ -45,6 +45,7 @@ class app extends Component {
 			numItems: 10,
 			items: imageItems
 		};
+		this.removeItem = createRef();
```

Comment:
since this new ref was created for removing the item, I think `this.rootRef` and `this.scrollingRef` are not needed and we can remove them and their usage in this file.